jwst/white_light/white_light_step.py

```diff
@@ -24,7 +26,7 @@ def process(self, input):
         with datamodels.open(input) as input_model:
 
             # Call the white light curve generation routine
-            result = white_light(input_model)
+            result = white_light(input_model,self.min_wavelength,self.max_wavelength)
```

Comment:
flake8 police: appropriate social/argument spacing.

Reply:
Unless you apply a mask ...